RUFAS/input_manager.py

```diff
@@ -789,3 +793,264 @@ def flush_pool(self) -> None:
                     }
         self.__pool = {}
         om.add_log("Clear variable pool", "The pool is emptied.", info_map)
+
+    def _metadata_properties_exists(self, variable_name: str, properties_blob_key: str) -> (bool, KeyError):
```

Comment:
```suggestion
    def _metadata_properties_exists(self, variable_name: str, properties_blob_key: str) -> (bool, KeyError | None):
```
Directly denoting that the error value might be None would be good because that is one of the expected cases.